server/api/orders.js

```diff
@@ -5,13 +5,14 @@ module.exports = router
 router.get('/', async (req, res, next) => {
   try {
     if (req.user) {
-      let order
-      order = await Order.findOne({
+      let order = await Order.findOne({
         include: {model: Product},
         where: {userId: req.user.id, status: 'inCart'}
       })
 
-      if (!order.id) {
+      await order.updateTotal()
+
+      if (!order) {
```

Comment:
I think we have to keep if(!order.id) from before, because if no match is found in Order.findOne, order={}, and that would not go into the if-statement.

Reply:
> I think we have to keep if(!order.id) from before, because if no match is found in Order.findOne, order={}, and that would not go into the if-statement.

Okay- let me see if I can change that back in my code and then do a new pull request.